### PR TITLE
`vyos.vyos` is no longer deprecated! 🎉 

### LINKED ISSUE #344: Removal from Ansible Community Package
There has been an announcement deprecating this collection in [The Bullhorn #123](https://forum.ansible.com/t/2568). It says there won't be any more major releases, but I don't see any minor or even bugfix releases either for quite some time.

This repo isn't archived and there is no information about deprecation neither in the repo nor on [galaxy](https://galaxy.ansible.com/ui/repo/published/vyos/vyos/). However, because of the deprecation announcement there is now a discussion about [removing this collecection from the Ansible Community Package](https://forum.ansible.com/t/6244).

At least one month after this announcement appears here and on [Bullhorn](https://forum.ansible.com/c/news/bullhorn/17), the Ansible Community Steering Committee will vote on whether this collection will be removed, or whether it will be kept. If it will be removed, this will happen earliest in Ansible 11. Please note that people can still manually install the collection with `ansible-galaxy collection install vyos.vyos` even when it has been removed from Ansible.

### PR DESCRIPTION
**Pull request by NilashishC**
##### SUMMARY
- Fixes #344
- Maintenance PR to allow vyos.vyos continue to be a part of Ansible Community Package.
- Updates all GHA workflows to use ansible-content-actions.
- 200+ ansible-lint fixes.
- Removes deprecation notice from README.

### CODE REVIEW

**Review by cidrblock — APPROVED** [review]

File: .isort.cfg [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [settings]
-known_first_party=ansible_collections.vyos.vyos
+known_first_party=ansible_collections.cisco.nxos
```
Comment (felixfontein) [inline .isort.cfg:2]: Is this change intentional?

File: bindep.txt [hunk 2]
```diff
@@ -2,8 +2,3 @@
 # see https://docs.openstack.org/infra/bindep/ for additional information.
 
 gcc-c++ [doc test platform:rpm]
-python3-devel [test platform:rpm]
-python3 [test platform:rpm]
-libssh-devel [test platform:rpm]
-libffi-devel [test platform:rpm]
-openssl-devel [test platform:rpm]
```
Comment (felixfontein) [inline bindep.txt:9]: Shouldn't these changes be mentioned in the changelog as well?

File: .isort.cfg (hunk 1 above)
Comment (NilashishC) [reply .isort.cfg]: Resolved.

File: .prettierignore [hunk 3]
```diff
@@ -12,6 +11,13 @@ env.bak/
...
 # A linked collection directory created by pytest-ansible-units
+
 collections/
 
+# Tracked but not manually edited
+
+# Tracked but manually formatted
+
+# WIP
+tests/integration/targets/nxos_config/templates/config.js
```
[the remark this answers is not in the document]
Comment (NilashishC) [reply .prettierignore]: Resolved.

File: bindep.txt (hunk 2 above)
Comment (NilashishC) [reply bindep.txt:9]: I believe it is expected to at least have python3 available on a container image that is to be used as an EE (built via builder). So, I'm not sure if this really warrants a changelog. The other ones are _only_ required for using pylibssh transport so I've added them back.

Comment (felixfontein) [reply bindep.txt:9]: I agree that `python3` shouldn't be in this list :)

**Review by gundalow — APPROVED** [review]
